docs/tutorials/tutorial-a-guide-to-using-the-snowplow-inspector-extension/index.md

```diff
@@ -0,0 +1,173 @@
+This registry is configured by default in most Snowplow Pipelines.
+By default, the extension will also create a "Local Registry", which lets you iterate on schemas without having to deploy them to a real registry.
+
+However the extension can be configured to compare events against private schema repositories (that contain your organization's proprietary schemas).
+If the extension is not configured to point towards your private schema repository then any custom events or entities will not be recognized by the extension and can not be automatically validated.
+
+Clicking on the “Unrecognised” button will present you with the “Edit Schema Registry” dialog (covered in the next section) to configure your own registry:
+
+![A Self Describing Event called 'example_event' is displayed in the Snowplow Inspector extension. It contains a property 'example_field_1' with the value 'abc'. The extension says the event's schema is Unrecognized. A cursor graphic is overlaid to indicate this label can be clicked.](images/unrecognized-event.png)
+![An image of the form used for configuring a custom Iglu Registry in the Snowplow Inspector extension.](images/edit-registry.png)
+
+Alternatively, your registries and schemas can be managed by using the ‘Manage Schemas’ button in the top toolbar.
+
+### Iglu registry and schema management
+To see a list of all the schemas supported by your currently configured registries, use the ‘Manage Schemas’ button in the top toolbar.
+The extension will connect to your registries and request a listing of all available schemas.
+The schemas are displayed on the left, and the configured registries on the right.
+
+![The Snowplow Inspector extension's Manage Schemas view is shown. Two Iglu registries are listed on the right with buttons for management. The main display has a searchable directory of schemas, grouped by vendor, name, format, and version.](images/manage-schemas.png)
+
+Selecting registries in the list will filter the schema listing to show only the schemas they contain.
+You can also search for schemas using the field at the top of the directory.
+Use the ‘Clear Filters’ button to clear these and see the full list of schemas again.
+
+Using the ‘Registries...’ button on the right, you can:
+
+- Add more custom registries
+- Edit existing registries you've previously configured
+- Remove configured registries so they aren't accessible to the extension any more
+- Import a list of registries from an existing [Iglu Resolver configuration](/docs/pipeline-components-and-applications/iglu/iglu-resolver/index.md)
+
+Selecting a "local" registry will activate the ‘Schemas...’ button and allow you to manage the schemas it contains.
+
+When adding or editing a registry, the following types are supported:
+1. [Local](#local-registries)
+1. [Data Structures API](#data-structures-api-registries)
```

Comment:
I know markdown supports this, but imho 1 2 3 4 reads nicer for those looking at the page source

Reply:
Fair! I think this is a bad habit I got from asciidoc because it lets me swaps lines around faster when editing. Updated.